frontend/app/component/error/Error.tsx

```diff
@@ -0,0 +1,56 @@
+import { useNavigate } from "react-router-dom";
+
+import { IconArrowLeft } from "@kiesraad/icon";
+import { Button, Icon } from "@kiesraad/ui";
+import { isDevelopment } from "@kiesraad/util";
+
+import cls from "./Error.module.css";
+import errorImage from "./error.png";
+import { ErrorAction } from "./Error.types";
+
+interface ErrorProps {
+  title: string;
+  children: React.ReactNode;
+  action: ErrorAction;
+  error?: Error;
+}
+
+export function Error({ title, error, action = ErrorAction.Back, children }: ErrorProps) {
+  const navigate = useNavigate();
+
+  return (
+    <>
+      <main>
+        <article className={cls.errorContainer}>
+          <section>
+            <h1>{title}</h1>
+            {children}
+            {action === ErrorAction.Back && (
+              <Button
+                size="lg"
+                variant="secondary"
+                onClick={() => {
+                  navigate(-1);
+                }}
+              >
+                <Icon icon={<IconArrowLeft />} size="sm" color="primary" />
```

Comment:
```suggestion
                <Icon icon={<IconArrowLeft />} size="md" color="primary" />
```
In the design the icon has a size of 1.5rem

Reply:
Fixed in 076879b4da23dbabdb80c84c7b3452896459e691